package-lock.json

```diff
@@ -1,12 +1,12 @@
 {
   "name": "shaka-player",
-  "version": "4.3.0",
+  "version": "4.4.0",
   "lockfileVersion": 2,
   "requires": true,
   "packages": {
     "": {
       "name": "shaka-player",
-      "version": "4.3.0",
+      "version": "4.4.0",
```

Comment:
Why? Revert this change

Reply:
I don't know how this happened. I'll revert.